src/actions/activities.js

```diff
@@ -0,0 +1,32 @@
+import uuid from 'uuid';
+
+export const addActivity = (
+  {
+    name = 'Just stuff',
+    teacher = 'Anonymous',
+    day,
+    classNo,
+    room = 'Anywhere'
+  } = {}
+) => ({
+  type: 'ADD_ACTIVITY',
+  activity: {
+    id: uuid(),
+    name,
+    teacher,
+    day,
+    classNo,
+    room
+  }
+})
+
+export const removeActivity = ({id} = {}) => ({
+  type: 'REMOVE_ACTIVITY',
```

Comment:
I recommend you to create a file with action names and use them instead of simple string. So:
`export const REMOVE_ACTIVITY = 'REMOVE_ACTIVITY'`. It is a good practice in redux, as js engine will tell you if you make a typo. In sttrings... not .

Also, but not in this PR (separate one) I recommend you to restucture the store files to have:
`store/activities/actions.js`
`store/activities/reducer.js`
`store/activities/filters.js`

Reply:
I see that `selectors` folder has been created. I don't see reason for `that.store/activities/actions.js`
`store/activities/reducer.js`
`store/activities/filters.js`
`store/activities/selectors.js`. 
BTW -> filters could be in selectors, but don't do taht in this PR